src/Spectre.Cli/FlagValue.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace Spectre.Cli
+{
+    internal interface IFlagValue
```

Comment:
Not sure about naming. Need something that makes it easy to understand what it does, but `FlagValue` might not be it.

Reply:
Yeah, I don't love `IFlagValue` - it suggests that it's a bool to me. Maybe `IOptionalValue`?

Reply:
But it is a bool. A bool (i.e. should we serve content) with an associated value (i.e. what port should we serve on?).